src/lib/draw-and-pick.js

```diff
@@ -71,8 +71,9 @@ export function pickLayers(gl, {
   // And compensate for pixelRatio
   const pixelRatio = typeof window !== 'undefined' ?
     window.devicePixelRatio : 1;
-  const deviceX = x * pixelRatio;
-  const deviceY = gl.canvas.height - y * pixelRatio;
+  const deviceX = Math.round(x * pixelRatio);
+  const deviceY = Math.round(gl.canvas.height - y * pixelRatio);
+  const deviceRadius = Math.round(radius * pixelRatio);
```

Comment:
In my local pass at this feature (you made it further than I did), I added `deviceW` / `deviceH`, to enable marquee selection. Is there any benefit to a circular selection instead of a rectangular one? What's the specific use case for circular other than "fat-finger"?